### PR TITLE
Migration to dotnet core 3.0

### PR DESCRIPTION
**Pull request by nicolas63**

### CODE REVIEW

**Comment by danielpetisme** [comment]
Thank you!!! I'll look at it quickly

File: generators/server/templates/dotnetcore/src/Project/Data/ApplicationDatabaseContext.cs.ejs [hunk 1]
```diff
@@ -17,6 +17,7 @@ using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
 
+
```
Comment (danielpetisme) [inline generators/server/templates/dotnetcore/src/Project/Data/ApplicationDatabaseContext.cs.ejs]: Can you remove the additional line, one line should be enough

File: generators/server/templates/dotnetcore/src/Project/Infrastructure/SwaggerStartup.cs.ejs [hunk 2]
```diff
@@ -14,14 +14,15 @@
...
+using Microsoft.OpenApi.Models;
 using Swashbuckle.AspNetCore.Swagger;
 
 namespace <%= namespace %>.Infrastructure {
     public static class SwaggerConfiguration {
         public static IServiceCollection AddSwaggerModule(this IServiceCollection @this)
         {
             @this.AddSwaggerGen(c => {
-                c.SwaggerDoc("v2", new Info {Title = "nhipsterSampleApplication API", Version = "0.0.1"});
+                c.SwaggerDoc("v2", new OpenApiInfo {Title = "nhipsterSampleApplication API", Version = "0.0.1"});
```
Comment (danielpetisme) [inline generators/server/templates/dotnetcore/src/Project/Infrastructure/SwaggerStartup.cs.ejs]: Can you take the opportunity to replace `nhipsterSampleApplication` by the application name variable please.

Comment (nicolas63) [reply generators/server/templates/dotnetcore/src/Project/Infrastructure/SwaggerStartup.cs.ejs]: Good idea

**Comment by danielpetisme** [comment]
I had a bounty on this issue, feel free to claim it, you deserve it!
https://www.jhipster.tech/bug-bounties/

thank you for your help

**Comment by danielpetisme** [comment]
fix #43

**Comment by nicolas63** [comment]
Bounty claimed https://opencollective.com/generator-jhipster/expenses/11024

**Pull request merged**